### ISSUE TITLE
Add sharing image to meta values

### ISSUE THREAD

**Issue by Eric-Arellano** [ParkingReformNetwork/reform-map#209]
We need to add an image to the `<header>`. See https://github.com/ParkingReformNetwork/parking-lot-map/blob/main/index.html for how this gets configured.

**Comment by tunglinn** [comment]
Not sure which image this is referring to. Although I see there might be something missing in `<head>` instead of `<header>`. 

Is it this part?
`<meta
      property="og:image"
      content="https://parkingreform.org/wp-content/uploads/2023/03/lotmapbanner.png"
    />`

**Comment by Eric-Arellano** [comment]
Exactly. I was thinking we should probably get a new image asset, but it's fine to reuse what we have now.
